rdr_service/model/database.py

```
@@ -24,6 +24,7 @@
 from rdr_service.model.deceased_report_import_record import DeceasedReportImportRecord
 from rdr_service.model.ehr import EhrReceipt, ParticipantEhrReceipt
 from rdr_service.model.hpo import HPO
+from rdr_service.model.hpro_consent_files import HealthProConsentFile
```

Codacy complaining about this with no way to rectify or ignore the error.